src/biotite/sequence/graphics/alignment.py

```diff
@@ -344,7 +344,142 @@ def get_color(self, alignment, column_i, seq_i):
         code = alignment.sequences[seq_i].code[index]
         return self._colors[code]
 
+    
+class ArrayPlotter(LetterPlotter):
+    '''
+    This SymbolPlotter quantitatively decorates sequences alignments, with molecular
+    recognition data obtaine from microarrays. Symbols are visualized as characters 
+    on a colored background box. The color of a given box represents the recognition 
+    signal. The intensity of the color, is proportional to the strenght of the 
+    recognition.
+    
+    Parameters
+    ----------
+    axes : Axes
+        A Matplotlib axes, that is used as plotting area.
+    fl_score : numpy.ndarray 
+        The ndarray to map recognition values to score residues.
```

Comment:
The ndarray itself does not map, right?

Reply:
It might be better to use the term "store".